options/index.js

```diff
@@ -3,58 +3,30 @@ import ReactDOM from 'react-dom'
 
 import '../common/hot-reload'
 
-const Form = () => {
+const Options = () => {
     const [webhookUrl, setWebhookUrl] = useState('')
+    const handleChange = (e) => setWebhookUrl(e.target.value) // TODO: handle url validation
```

Comment:
TBH, I'm not sure what URL validation will do here. I added it because I know it's just good practice to validate/sanitize any user input. But none of this data is going to a service we own, as everything is contained on the user's machine. Can you think of how a user or website could "hack" the extension and cause some harm by providing some bad inputs?

Reply:
I'm not sure what harm could come of it but you never know! We'll add some simple regex validation in a fun React-y way at some point.